src/mono/wasm/Wasm.Build.Tests/Blazor/BlazorWasmProjectProvider.cs

```diff
@@ -28,4 +29,14 @@ public void AssertBundle(BlazorBuildOptions options)
                 AssertIcuAssets: true,
                 AssertSymbolsFile: false // FIXME: not supported yet
             ));
+    
+    public override string FindBinFrameworkDir(string config, bool forPublish, string framework, string? projectDir = null)
```

Comment:
Why do we need to override it here? Blazor and Wasm SDK should behave the same

Reply:
That's true, originally I didn't notice they are the same, it can be left in `WasmSdkBasedProjectProvider` only.
Note for the future: once we will align `BlazorBuildOptions` and `AssertWasmSdkBundleOptions` then maybe we will even use WASM SDK provider in Blazor and drop `BlazorWasmProjectProvider`.